resources/qml/dialogs/ImagePackEditorDialog.qml

```diff
@@ -61,6 +61,7 @@ ApplicationWindow {
                 header: AvatarListTile {
                     title: imagePack.packname
                     avatarUrl: imagePack.avatarUrl
+                    userid: imagePack.packname
```

Comment:
You should probably set the roomid here instead, since you never want an online indicator. And use the statekey.

Reply:
Oooh yes, good thought. What's the statekey anyway? Sort of an mxid for an image pack?